docs/topics/gradle.md

```diff
@@ -975,16 +975,16 @@ _Kotlin compiler execution strategy_ defines where the Kotlin compiler is execut
+|----------------|--------------------------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Daemon         | Inside its own daemon process        | Yes                     | *The default strategy*. Can be shared between different Gradle daemons                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| In process     | Inside the Gradle daemon process     | No                      | May share the heap with the Gradle daemon. Note that the in process execution is *slower* than the Daemon execution strategy. This is so because Gradle spawns multiple [workers](https://docs.gradle.org/current/userguide/worker_api.html) with the Kotlin compiler, and they could be not yet [JIT-optimized](https://en.wikipedia.org/wiki/Just-in-time_compilation#:~:text=In%20computing%2C%20just%2Din%2D,time)%20rather%20than%20before%20execution.) when the compilation runs. Also, the Kotlin daemon has additional caches. |
+| Out of process | In a separate process for each call  | No                      | -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Probably worth to explain as well:

> Same as `in-process`, but instead of using Gradle workers - each compilation will run in a separate Java process. The slowest of the three execution strategies

Reply:
> each compilation will start and run in a separate Java process

We already have it in the second column:
> In a separate process for each call

Is it really worth mentioning here that it is
> instead of using Gradle workers
?

This is the inner kitchen of it, in the forth column for the "In process" we mention workers in terms of JIT optimization and as one of the reasons why this strategy is slow. Not because of a user really needs to know what's under the hood.

Reply:
And I wanted to consult both of you: won't we frighten users telling that these strategies are slow? Won't it make them using only the default strategy? cc @ALikhachev

Reply:
Ok, seems like we can write that they are slow, and it's all a user's responsibility to use slower strategies knowing this fact about them, yeah?

Reply:
Seems like I can resolve this?